README.md

```diff
@@ -61,6 +61,8 @@ The source code can be found at https://github.com/kiwix/kiwix-js
 
 Unit tests can be run by opening `tests/index.html` file in Firefox, Edge, or Chromium/Chrome.
 
+Before running the tests, a one-time set up is needed to fetch development dependencies from the npm registry. Run `npm ci --ignore-scripts` to fetch the same versions as we use in CI.
```

Comment:
I presume the dev also needs to run `npm install` before `npm ci --ignore-scripts`? If that's correct, then for the sake of newbie devs (and we do get quite a lot of undergraduate compsci students wanting to get PR experience), maybe add that to this instruction?

Reply:
No, it's better to keep only `npm ci` (provided that npm version is >=6). See https://medium.com/better-programming/npm-ci-vs-npm-install-which-should-you-use-in-your-node-js-projects-51e07cb71e26

Reply:
Indeed, as @mossroy says.

`npm ci` is a stricter variant of `npm install` that fully adheres to the recursive package-lock file and prefers local caches, instead of freshly computing and calculating direct and indirect dependency version ranges.

It is an unfortunate turth that `npm ci` does what what one would reasonably expect `npm install` to do, and `npm install` does effectively what one would expect `npm update` to do.

The only difference to me is that `npm update` also saves the implied updates to `package.json`. In other words, when package.json specifies `foo: ^1.2.3`, npm install could install 1.5.5 which is compatible, whereas  `npm update` would install 1.5.5 and then also update package.json to `^1.5.5`.